src/services/billing/cost-management/cost-analysis/modules/CostAnalysisChart.vue

```diff
@@ -207,26 +241,20 @@ export default {
                         category: groupByName,
                         value: d.usd_cost,
                     });
-                    groupByNameSet.add(groupByName);
                 });
-            } else if (rawData[0]?.usd_cost) {
+            } else if (rawData.length) {
                 chartData = [{
                     category: 'Total Cost',
-                    value: rawData[0].usd_cost,
+                    value: rawData[0]?.usd_cost || 0,
```

Comment:
이건 그냥 마이너한건데, 0도 결국 false이기 때문에, 이 경우엔 null & undefined만 체크하는 nullish 병합 연산자(`??`)를 쓰는게 더 좋아보입니다 😄

```suggestion
                    value: rawData[0]?.usd_cost ?? 0,
```